docassemble/ALAutomatedTestingTests/data/questions/all_tests.yml

```diff
@@ -9,50 +9,10 @@ default screen parts:
   post: |
     <div data-variable="${ encode_name(str( user_info().variable )) }" id="trigger" aria-hidden="true" style="display: none;"></div>
 ---
-# Do not edit this block
-comment: |
-  # Fields to include
-  
-  ## Regular:
-  - [x] Radio (yes/no)
-  - [x] Radio (other)
-  - [x] Checkbox (none of)
-  - [x] Checkbox (yes/no)
-  - [x] Checkbox (other)
-  - [x] Buttons (yes/no)
-  - [x] Buttons (other)
-  - [x] Buttons (continue)
-  - [x] Dropdown
-  - [x] Text area
-  - [x] Text input (type text)
-  - [x] Text input (type date)
-  
-  ## Showif:
-  - [x] Radio (yes/no)
-  - [x] Radio (other)
-  - [x] Checkbox (none of)
-  - [x] Checkbox (yes/no)
-  - [x] Checkbox (other)
-  - [x] Dropdown
-  - [x] Text area
-  - [x] Text input (type text)
-  
-  # Other things to include:
-  - [x] Terms
-  - [x] Button that's a link
-  - [x] A link
-  - [x] Button that's not a continue button
-
-  # Situations to test:
-  - [ ] Continue button field is listed in the table before other unrequired fields on the page (for the story table)
-  - [ ] Test not filling in an unrequired field?
```

Comment:
Just to clarify - do we have tickets for these two unimplemented items, or are these not relevant anymore?

Reply:
Good question and those tests did get added.